ebr_connector/prepacked_queries/test_results.py

```diff
@@ -0,0 +1,94 @@
+"""
+A collection of queries that provide failing test results
```

Comment:
I think we should remove "failing" here, as this module will be expanded for general test result queries